js/packages/openinference-instrumentation-llama-index/test/llamaIndex.test.ts

```diff
@@ -320,3 +338,136 @@ describe("LlamaIndexInstrumentation", () => {
     });
   });
 });
+
+describe("LlamaIndexInstrumentation - LLM", () => {
+  const memoryExporter = new InMemorySpanExporter();
+  const spanProcessor = new SimpleSpanProcessor(memoryExporter);
+  instrumentation.setTracerProvider(tracerProvider);
+
+  tracerProvider.addSpanProcessor(spanProcessor);
+  // @ts-expect-error the moduleExports property is private. This is needed to make the test work with auto-mocking
+  instrumentation._modules[0].moduleExports = llamaindex;
+
+  beforeAll(() => {
+    instrumentation.enable();
+
+    // Setup the OPENAI_API_KEY
+    process.env["OPENAI_API_KEY"] = "fake-api-key";
+  });
+  afterAll(() => {
+    instrumentation.disable();
+    // Delete the OPENAI_API_KEY
+    delete process.env["OPENAI_API_KEY"];
+  });
+  beforeEach(() => {
+    memoryExporter.reset();
+
+    jest.spyOn(OpenAI.Chat.Completions.prototype, "create").mockImplementation(
+      // @ts-expect-error the response type is not correct - this is just for testing
+      async (): Promise<unknown> => {
+        return CHAT_RESPONSE;
+      },
```

Comment:
just curious is there a way to make it correct - totally fine to fallback to this if it's too much to fully mock, but if it's a fairly simple response we should try to get it right